react-nesting/react-nesting_fragments/src/App.jsx

```diff
@@ -1,13 +1,12 @@
 import React from "react";
 import "./styles.css";
 import Box from "./components/Box";
+import Boxes from "./components/Boxes";
 
 export default function App() {
   return (
     <main className="flex-container">
-      <Box color="#007bff" />
-      <Box color="#fc3" />
-      <Box color="#ff3333" />
+      <Boxes></Boxes>
```

Comment:
```suggestion
      <Boxes/>
```
This works but it also implies potential children. The self closing form is cleaner, easier to scan, and reduces the chance someone accidentally dropping whitespace or invisible children inside :-)